Jessy/playlist_challenge.py

```diff
@@ -0,0 +1,61 @@
+# Music playlist challenge!
+# In this challenge, you'll have to both work on this script, and the accompanying script playlist_functions.py
+# All of your functions should be in the other script (playlist_functions.py)
+# But, you'll run your functions here
+
+# 1 Import all the functions in playlist_functions.py
+from playlist_functions import *
+from numpy import *
+
+# This code initializes your playlist as an empty list. no songs in it yet!
+my_playlist = []
+
+
+# 2 Check what is in your playlist using the display_playlist() function
+display_playlist(my_playlist)
+# HINT: the display_playlist() function in playlist_functions.py to figure out how to use it
+print('Question 2')
+
+# 3 Add a song to my_playlist using the add_song() function
+add_song(my_playlist, {"artist":"Tupac","title":"To Live And Die In LA"})
+# The song that you add should be a dictionary, with the following key-value pairs
+# 'artist' (string)
+# 'title' (string)
+
+'''
+example_song = {'artist': 'Lauryn Hill', 'title': 'Everything Is Everything'}
+'''
+
+
+# 4 Check that you've added the song by running the display_playlist() function again
+display_playlist(my_playlist)
+print('Question 4')
+
+# 5 Add 2 more songs to my_playlist, then display it again using the display_playlist() function
+print('Question 5')
+add_song(my_playlist, {'artist':'Too Short', 'title':'The Ghetto'})
+add_song(my_playlist, {'artist':'Thug Life', 'title':'Cradle to the Grave'})
+display_playlist(my_playlist)
+
+# 6 In playlist_functions.py, define a function called get_playlist_length()
+
+# See playlist_functions.py for details on how to define this function
+# THEN, call that function in this script to get the length of my_playlist
+
+print('Question 6')
+from  playlist_functions import *
```

Comment:
The 'import *' at the top of your script covers all of your code. As as long as the file/module where you are importing from is saved, everything added there will be imported.